media/js/firefox/dev-firstrun.js

```diff
@@ -25,8 +25,7 @@ function onYouTubeIframeAPIReady() {
     var $window = $(window);
     var $document = $(document);
     var TARGET_1 = 'devtools';
-    var TARGET_2 = 'webide';
-    var TARGET_3 = 'appMenu';
+    var TARGET_3 = 'appMenu'; // TARGET_2 was WebIDE, since removed, so we're skipping a number here rather than renaming everything.
```

Comment:
The JS changes here all look good to me, I tested this locally and works fine :smile: